tests/integration/json_test.go

```diff
@@ -34,6 +34,22 @@ var _ = Describe("odojsonoutput", func() {
 		os.Unsetenv("GLOBALODOCONFIG")
 	})
 
+	Context("odo machine create project json output", func() {
+		// odo project create newprojectjson -o json
+		It("should be able to create project and show output in json format", func() {
+			actual := helper.CmdShouldPass("odo", "project", "create", "newprojectjson", "-o", "json")
+			desired := `{"kind":"Project","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"newprojectjson","namespace":"newprojectjson","creationTimestamp":null},"message":"Project 'newprojectjson' is ready for use"}`
+			Expect(desired).Should(MatchJSON(actual))
+		})
+
+		// odo project create newprojectjson -o json (x2)
+		It("should fail saying that there is already an existing project", func() {
+			actual := helper.CmdShouldFail("odo", "project", "create", "newprojectjson", "-o", "json")
+			desired := `{"kind":"Error","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"creationTimestamp":null},"message":"unable to create new project: unable to create new project newprojectjson: project.project.openshift.io \"newprojectjson\" already exists"}`
```

Comment:
```desired := `...project.project.openshift.io \"newprojectjson\" already exists"}```

These two back slashes are unnecessary i guess.

Reply:
No they're required (they pass on Travis), it's the parallel run which are messing it up..

Reply:
@cdrage i am concern about the message format ``` \"test\" ```.  for example

$ odo project create test -o json
{"kind":"Error","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"test","creationTimestamp":null},"message":"unable to create new project: unable to create new project test: project.project.openshift.io **\"test\"** already exists"}

Reply:
Yeah, this is normal from OpenShift's output :+1: We do this automatically in json marshalling since this is the output coming from the OpenShift cluster.

Reply:
Thanks for the details